tests/testthat/test_mod_mappingColumn.R

```diff
@@ -4,10 +4,12 @@ library(shinytest)
 library(testthat)
 library(stringr)
 
+skip_on_cran()
```

Comment:
Are we having issues with tests on CRAN? I know this is becoming more popular and I generally agree with it but just curious.

Reply:
phantomjs isn't set up, so `shinytest` was bombing. There might be a workaround, but nothing obvious with a little googling.